azurerm/helpers/authentication/environment.go

```diff
@@ -1,6 +1,27 @@
 package authentication
 
-import "strings"
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Azure/go-autorest/autorest/azure"
+)
+
+func DetermineEnvironment(name string) (*azure.Environment, error) {
```

Comment:
Nit: Exported functions should be documented

Reply:
done 👍